.gitignore

```
@@ -13,3 +13,4 @@ igraph.Rcheck/
 .Rproj.user
 /configure~
 /build/
+docs
```

Let's be precise here:

```suggestion
/docs/
```